.travis.yml

```diff
@@ -50,8 +51,9 @@ cache:
     - $HOME/.gradle/wrapper/
 script:
   - travis_wait ./gradlew installJBrowseWebOnly gwtc installJBrowseTools && ./grailsw refresh-dependencies --stacktrace && ./grailsw test-app -coverage -xml --stacktrace
-  #- node web-app/jbrowse/src/dojo/dojo.js load=build --require "web-app/jbrowse/src/JBrowse/init.js" --profile "web-app/jbrowse/plugins/WebApollo/js/WebApollo"
   - jshint client/apollo/js
+  - ./grailsw run-app & && cd python-apollo && export ARROW_GLOBAL_CONFIG_PATH=`pwd`/test-data/arrow.yml && ./bootstrap_apollo.sh && python setup.py nosetests
```

Comment:
Shoot, didn't see this line, so sorry @nathandunn, my suggestion was bad if you need to execute those tests, I should have read more closely